### PR TITLE
#24864: adding slack message notification logic to individual developer

### PR DESCRIPTION
**Pull request by victoralfaro-dotcms**

### CODE REVIEW

**Review by freddyDOTCMS — APPROVED** [review]

File: .github/data/slack-mappings.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    "github_user": "hmoreras",
+    "slack_id": "U6YCPDC1Z"
+  },
+  {
+    "github_user": "hmoreras",
+    "slack_id": "U6YCPDC1Z"
+  },
+  {
+    "github_user": "hmoreras",
+    "slack_id": "U6YCPDC1Z"
```
Comment (fabrizzio-dotCMS) [inline .github/data/slack-mappings.json:44]: This is a dupe. And there are folks in the list that no longer work here

**Review by fabrizzio-dotCMS — DISMISSED** [review]

File: .github/data/slack-mappings.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  {
+    "github_user": "bryanboza",
+    "slack_id": "U028VTVU8"
+  },
+  {
+    "github_user": "cobbg",
+    "slack_id": "U018GD00WLQ"
+  },
+  {
+    "github_user": "hmoreras",
```
Comment (dsilvam) [inline .github/data/slack-mappings.json:35]: Dup here also

File: .github/data/slack-mappings.json [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    "github_user": "fabrizzio-dotCMS",
+    "slack_id": "U9J1063LG"
+  },
+  {
+    "github_user": "fmontes",
+    "slack_id": "U028Y2YH0"
+  },
+  {
+    "github_user": "hmoreras",
+    "slack_id": "U6YCPDC1Z"
```
Comment (dsilvam) [inline .github/data/slack-mappings.json:60]: Dup

File: .github/data/slack-mappings.json [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  {
+    "github_user": "oidacra",
+    "slack_id": "U034YTRU4MB"
+  },
+  {
+    "github_user": "hmoreras",
+    "slack_id": "U6YCPDC1Z"
+  },
+  {
+    "github_user": "alfredo-dotcms",
```
Comment (dsilvam) [inline .github/data/slack-mappings.json:23]: remove

File: .github/data/slack-mappings.json (hunk 2 above)
Comment (jdotcms) [reply .github/data/slack-mappings.json:35]: is a bless to have two betos

File: .github/data/slack-mappings.json (hunk 3 above)
Comment (jdotcms) [reply .github/data/slack-mappings.json:60]: it seems we have many beto's all good

**Review by jdotcms — APPROVED** [review]

**Review by spbolton — APPROVED** [review]

File: .github/data/slack-mappings.json (hunk 2 above)
Comment (dsilvam) [reply .github/data/slack-mappings.json:31]: actually like five or six :P

**Review by dsilvam — APPROVED** [review]

**Pull request merged**